src/Providers/MetadataControllerProvider.php

```diff
@@ -37,8 +37,15 @@ public function boot()
         $ends = array();
         $Classes = $AutoClass::$classMap;
         foreach ($Classes as $name => $file) {
-            if (in_array("AlgoWeb\\PODataLaravel\\Controllers\\MetadataControllerTrait", class_uses($name))) {
-                $ends[] = new $name();
+            try {
```

Comment:
If class_uses call loses plot for whatever reason, then we don't want to register it as a controller endpoint